source/opt/ir_context.h

```diff
@@ -1208,6 +1212,25 @@ Instruction* IRContext::GetMemberName(uint32_t struct_type_id, uint32_t index) {
   return nullptr;
 }
 
+void IRContext::CloneNames(uint32_t old_id, uint32_t new_id, uint32_t new_len) {
```

Comment:
personal nit: would change this to something like "max_member_index", and use UINT32_MAX as default value, as It could be useful to not copy any member name.

Reply:
Done.